index.html

```diff
@@ -48,17 +48,13 @@ <h1 > hey there! <br/>
 			<div class="about_me_box">
 				<div class="about_me">
 					<h3>About me</h3>
-					<p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-						labore et dolore magna aliqua. Velit ut tortor pretium viverra suspendisse. Amet nulla facilisi
-						morbi tempus. Ac turpis egestas sed tempus urna et pharetra.
-						Pharetra vel turpis nunc eget lorem dolor sed viverra ipsum. Accumsan in nisl nisi scelerisque.
-						Aliquam etiam erat velit scelerisque in. Penatibus et magnis dis parturient montes nascetur.
-						Lobortis feugiat vivamus at augue eget arcu.<br /> Placerat duis ultricies lacus sed turpis
-						tincidunt id. Eu lobortis elementum nibh tellus molestie nunc non blandit. Aliquet nec
-						ullamcorper sit amet risus. Vitae tortor condimentum lacinia quis vel eros donec. Adipiscing
-						elit duis tristique sollicitudin nibh sit. Ac felis donec et odio. </p>
+					<p> Hi, I&#39m Hannah! After many years working as a Graphic Designer, I decided to make a career change (an idea both exciting and slightly terrifying!). I&#39m currently training to be a full stack developer at Mayden Academy in Bath, a 16 week intensive course to learn the skills required for the industry.
```

Comment:
More of a formatting point than anything else, but you might want to split this across multiple lines for the sake of readability

Reply:
Also on this line - the validator doesn't like the way you've escaped this character - it seems to want you to put a semicolon immediately before the m